projects/storefrontlib/shared/components/chat-messaging/messaging/messaging.model.ts

```diff
@@ -0,0 +1,28 @@
+export interface MessageDetails {
+  messages: Array<MessageEvent>;
+}
+
+export interface MessageEvent {
+  author?: string;
+  addedByAgent?: boolean;
+  createdAt?: string;
+  message?: string;
+  attachments?: Array<Attachment>;
+}
+
```

Comment:
It is too coupled in concept with Ticketing Event.
How about:
```suggestion
export interface Message {
  author?: string;
  leftAlign?: boolean;
  createdAt?: string;
  text?: string;
  attachments?: Array<Attachment>;
}

```

Reply:
yup, seems more generic.